api/src/services/assets.ts

```diff
@@ -122,6 +122,9 @@ export class AssetsService {
 		const type = file.type;
 		const transforms = transformation ? TransformationUtils.resolvePreset(transformation, file) : [];
 
+		const modified_on = file.modified_on ? new Date(file.modified_on) : undefined;
+		const version = modified_on ? (modified_on.getTime() / 1000).toFixed() : undefined;
```

Comment:
Let's use camelCase instead and we'll need a changeset too.

```suggestion
		const modifiedOn = file.modified_on ? new Date(file.modified_on) : undefined;
		const version = modifiedOn ? (modifiedOn.getTime() / 1000).toFixed() : undefined;
```